src/callkit.cs

```diff
@@ -32,6 +32,7 @@ public enum CXErrorCode : long {
 		Unknown = 0,
 		Unentitled = 1,
 		InvalidArgument = 2,
+		MissingVoIPBackgroundMode = 3
```

Comment:
minor: end new enum members with `,`
it's valid C# and it means the next update won't create a larger diff (since your line won't have to be modified)